README.md

```diff
@@ -66,79 +66,6 @@ More details on available commands can be found by running:
 make
 ```
 
-# Legacy developer set up instructions
```

Comment:
Were you able to run through the docker setup and have everything work? That'd be great if so.

Reply:
Yeah! It works great!